go-runtime/ftl/option.go

```diff
@@ -223,3 +223,25 @@ func (o *Option[T]) Unmarshal(
 	o.ok = true
 	return nil
 }
+
+// SetNull sets the Option to None
+func (o *Option[T]) SetNull() {
```

Comment:
This would also be unnecessary, as Option implements the SQL scanner/valuer interfaces.